packages/cli/src/lib/templates/dotredwood.md.template

```diff
@@ -0,0 +1,37 @@
+# .redwood
+
+## What is this directory?
+Redwood uses this `.redwood` directory to store transitory data that aids in the smooth and convienent operation of your Redwood project.
+
+## Do I need to do anything with this directory?
+No. You shouldn't have to create, edit or delete anything in this directory in your day-to-day work with Redwood.
+
+You not need to commit this directory to your version control system. It is ignored by default.
+
+## What should be in this directory?
+
+### Files
+
+| Name              | Description |
+| :---------------- | :------- |
+| commandCache.json | This file contains mappings to assit the redwood CLI in efficently executing commands. |
+| schema.graphql    |  |
+| studio.db         | The sqlite database used by the experimental `rw exp studio` feature. |
+| telemetry.txt     | Contains a unique ID used for telemetry. This value is rotated every 24 hours to protect your project's anonymity. |
+| test.db           | An sqlite database used when running tests |
```

Comment:
```suggestion
| test.db           | A sqlite database used when running tests. |
```

Reply:
Are you sure Dom? I'd say "An sqlite database" as in "An Ess-Que-Lite database"

Reply:
Oh good point, I pronounce it my head like "sequel" so that's why I thought "a". But "SQL" is also totally valid. So not sure, both are right and wrong I guess haha

Reply:
English is an interesting language 🇬🇧🇺🇸
"A RW project" or "An RW project"?

Reply:
"The sqlite database..." fixes this